docs/tls.md

```diff
@@ -10,7 +10,7 @@ To enable automatic CSR generation on Tenant, set `requestAutoCert` field in the
 
 - commonName: By default this is set to a wild card domain name as per [Kubernetes StatefulSet Pod Identity](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-identity). Set it any other value as per your requirements.
 
-- organizationName: By default set to `system:node`. Change it to the name of your organization.
+- organizationName: By default set to list `[system, node]`. Change it to the name of your organization. Any value added under this section will be appended to the list of existing organization names.
```

Comment:
```suggestion
- organizationName: By default set to list `["system:node"]`. Change it to the name of your organization. Any value added under this section will be appended to the list of existing organization names.
```

we do set the default to `system:nodes`

Reply:
Good catch; I was wrong about the name being appended too.